plans/PHASE10_CURATION.md

```
@@ -413,82 +420,70 @@ Tests 2 and 3 only pass once **both** of these are true:
+program's error anchors exist, run the WS-5 calibration and the tiers
+should align.
 
 ---
 
 ## How do we get the confidence + calibration data?
 
-Per `AUTONAV_PLAN.md` Part 10 §"Confidence calibration", the formula
-for every technique is:
+The formula for every technique is:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)